modules/deploy/pages/setting-up-dr-cluster.adoc

```diff
@@ -40,33 +41,67 @@ It serves an important role in supporting Disaster Recovery (DR) and Data Migrat
 
 == Recommended Deployment Models
 
+
+=== Zero Downtime Active-Active Disaster Recovery
+// tag::zero-downtime-active-active[]
+
+This model provides true zero-downtime disaster recovery using bi-directional XDCR between two active mobile clusters.
+Both clusters remain operational, with no fail-over through load balancer switching.
+You must configure both clusters with `shared_bucket_access=true` and `import_docs=true`.
+
+
+.Set Up
+To set up zero-downtime disaster recovery:
+
+. Configure bi-directional XDCR between the Primary and disaster recovery clusters. +
+Enable automatic filtering of cluster specific metadata.
+. Deploy Sync Gateway in active mode on both clusters.
```

Comment:
Documents and attachment are replicated with XDCR but users, roles, and databases are not replicated.

These need to be configured independently on both clusters.

Reply:
Thank you. I've updated to reflect this.